moment-range/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Moment.js 2.0.4
+// Type definitions for Moment.js 2.17.1 && Moment-range.js 3.0.3
```

Comment:
this is not a valid verison. the publisher uses the version from the header to set the version on the `@types/moment-range` package. 

the version here should match the `major.minor` of the corresponding package, i.e `3.0`